src/sql/parts/profiler/editor/profilerEditor.ts

```diff
@@ -397,16 +397,14 @@ export class ProfilerEditor extends BaseEditor {
 			return;
 		}
 
-		if (e.isRunning) {
-			this._startAction.enabled = !this.input.state.isRunning;
+		if(e.isPaused){
```

Comment:
could you please add a ' ' character between the 'f' and '(' characters in this if statement to follow the project code formatting guidelines?  Same comment for `if` at https://github.com/Microsoft/sqlopsstudio/pull/1615/files#diff-8db7555fa4f9d1bbc7f9a162528f47aeR104.